src/features/collaborator-avatars/__tests__/CollaboratorAvatarItem.test.js

```diff
@@ -1,9 +1,12 @@
 import * as React from 'react';
 
-import CollaboratorAvatarItem from '../CollaboratorAvatarItem';
+import { CollaboratorAvatarItemBase as CollaboratorAvatarItem } from '../CollaboratorAvatarItem';
 
 describe('features/collaborator-avatars/CollaboratorAvatarItem', () => {
-    const getWrapper = (props = {}) => shallow(<CollaboratorAvatarItem id={1} name="foo bar" {...props} />);
+    const intl = {
```

Comment:
minor: could we move this above the `describe`? that was the pattern we've been doing for the other ones